src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Implementation/AppBase.cs

```diff
@@ -299,6 +302,62 @@ public virtual async Task<List<Receiver>> GetEFormidlingReceivers(Instance insta
             return new List<Receiver> { receiver };
         }
 
+        /// <inheritdoc />
+        public async Task UpdatePresentationTexts(Instance instance, string dataType, object oldData, object updatedData)
+        {
+            int partyId = int.Parse(instance.InstanceOwner.PartyId);
+            Guid instanceGuid = Guid.Parse(instance.Id.Split("/")[1]);
+
+            if (_appMetadata.PresentationFields != null && _appMetadata.PresentationFields.Any(pf => pf.DataTypeId == dataType))
+            {
+                Dictionary<string, object> changes = null;
+
+                try
+                {
+                    changes = JsonHelper.FindChangedFields(JsonConvert.SerializeObject(oldData), JsonConvert.SerializeObject(updatedData));
+                }
+                catch (Exception e)
+                {
+                    // should a failure in finding changed fields crash the app?? 
```

Comment:
should a failure in finding changed fields crash the app??

Reply:
I'm leaning towards "yes". I would like errors related to this to be discovered as early as possible. Catching the exceptions might hide issues that would have otherwise been discovered during development.

Reply:
catching exception removed